spec/dummy/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
```

Comment:
I've stolen the majority of [spec/dummy/application.rb](https://github.com/thoughtbot/ember-cli-rails/blob/a6584138ebc37a5b029d5a7a723ad4362384bd0f/spec/dummy/application.rb) from [Clearance](https://github.com/thoughtbot/clearance/blob/7a63a2a9363729c743693d74f725337b860ee314/spec/dummy/application.rb)